tests/adiabatic_conditions/screen-output

```diff
@@ -23,9 +23,9 @@ Number of degrees of freedom: 40,836 (25,090+3,201+12,545)
      Rayleigh number:                               0
 
      Writing graphical output:           output-adiabatic_conditions/solution/solution-00000
-     RMS, max velocity:                  2.81e+04 m/s, 1.12e+05 m/s
+     RMS, max velocity:                  1.24e+04 m/s, 1.37e+05 m/s
      Temperature min/avg/max:            1613 K, 1613 K, 1613 K
-     Heat fluxes through boundary parts: 6721 W, -99.27 W, 0 W, 0 W
+     Heat fluxes through boundary parts: -4112 W, 1.907 W, 0 W, 0 W
```

Comment:
These are kind of large differences, though...

Reply:
but this test has an unreasonable solution anyway, the max velocity is 137 km/s, that would throw the material out of our solar system ;-). I opened #6639 to hopefully fix this. I remember this test raising flags in a previous deal.II update already.